Regression/Checksum/benchmarks_json/ElectrostaticSphere.json

```diff
@@ -1,17 +1,17 @@
 {
   "lev=0": {
-    "Ex": 6.504803536004636,
-    "Ey": 6.504803536004637,
-    "Ez": 6.504803536004637,
+    "Ex": 6.504631859049922,
```

Comment:
I notice this file and `ElectrostaticSphereRelNodal.json` match exactly. In a different PR we could make a single analysis script that would use the same json file for both tests.